numpy/fft/_pocketfft.py

```diff
@@ -160,7 +160,7 @@ def fft(a, n=None, axis=-1, norm=None):
     Raises
     ------
     IndexError
-        if `axes` is larger than the last axis of `a`.
+        if `axis` is larger than the last axis of `a`.
```

Comment:
"larger" is also ambiguous. Maybe "If `axis` is not an axis of `a`.

Reply:
Sure; commit amended and force pushed; that updates 3 other docstrings that had "`axis`" right but were using the same phrasing.